From 6febf70e657922cf55319f7331b48af9f89ac14d Mon Sep 17 00:00:00 2001
From: Sam Nazarko <email@samnazarko.co.uk>
Date: Tue, 7 Feb 2017 17:59:15 +0000
Subject: [PATCH] [package] [mediacenter-osmc] Ensure that BTPlayer add-on is
 always enabled. This allows the add-on to be enabled, even if it is installed
 later.

Signed-off-by: Sam Nazarko <email@samnazarko.co.uk>
---
 xbmc/addons/AddonManager.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/xbmc/addons/AddonManager.cpp b/xbmc/addons/AddonManager.cpp
index c86c6e9..f6e593d 100644
--- a/xbmc/addons/AddonManager.cpp
+++ b/xbmc/addons/AddonManager.cpp
@@ -814,6 +814,9 @@ bool CAddonMgr::IsAddonDisabled(const std::string& ID)
   std::regex addons_disable("plugin.program.xbmchub.notifications");
   if (regex_match(ID, addons_disable))
     return true;
+  std::regex addons_enabled("service.osmc.BTPlayer");
+  if (regex_match(ID, addons_enabled))
+    return false;
   return m_disabled.find(ID) != m_disabled.end();
 }
 
-- 
2.7.4

